### ISSUE TITLE
Panel loader not working with remote DCOS Cassandra

### ISSUE THREAD

**Issue by jmarin** [cfpb/hmda-platform#1023]
The panel loader is not working properly when trying to load data into the `DCOS` cluster. The stack trace is as follows:

```
[info] [DEBUG] [06/28/2017 12:12:51.988] [main] [EventStream] StandardOutLogger started
[info] [DEBUG] [06/28/2017 12:12:52.339] [main] [EventStream(akka://hmda)] logger log1-Logging$DefaultLogger started
[info] [DEBUG] [06/28/2017 12:12:52.341] [main] [EventStream(akka://hmda)] Default Loggers started
java.lang.RuntimeException: Nonzero exit code returned from runner: 3
	at scala.sys.package$.error(package.scala:27)
[trace] Stack trace suppressed: run last panel/compile:run for the full output.
[error] (panel/compile:run) Nonzero exit code returned from runner: 3
[error] Total time: 4 s, completed Jun 28, 2017 12:12:53 PM
>
```
A single API call with an institution in JSON format succeeds and stores the institution data into the `Cassandra` backend, which means that the API is working fine. FWIW I have been able to use the panel loader locally against a `Cassandra` docker container

**Comment by nickgrippin** [comment]
Given that error code, it looks like the call to `/institutions/create` didn't return properly.  I'll look into it.

**Comment by nickgrippin** [comment]
That new admin endpoint was merged into master 5 days ago and will be released in `1.8`.  Currently, the `1.7` version deployed to the cluster doesn't have that endpoint, so the panel parser will stop when it can't create the Institutions schema.  

This can be fixed by either **a)** waiting until the Friday release before trying to run the panel loader, or **b)** deploying the master branch to the cluster

**Comment by nickgrippin** [comment]
Closing, panel correctly loaded data into the cluster as of the 1.8 release.

There's an issue with the Institution ID that will be fixed with #1026
